### PR TITLE
Comments about btstack_run_loop_execute

### PR DESCRIPTION
**Pull request by peterharperuk**
Add some comments about how btstack_run_loop_execute does nothing in threadsafe backrgound mode.

### CODE REVIEW

File: pico_w/bt/standalone/client.c [hunk 1]
```diff
@@ -268,6 +268,15 @@ int main() {
     // turn on!
     hci_power_control(HCI_POWER_ON);
 
+    // We're using threadsafe background where poll does nothing and work is performed in a low priority IRQ
+    // So btstack_run_loop_execute effectively does nothing and you're free to just loop and do your own stuff
+    // You would have to call btstack_run_loop_execute if you were using pico_cyw43_arch_poll
+#if 1
```
Comment (lurch) [inline pico_w/bt/standalone/client.c:274]: Apologies for the stupid question, but why does this PR add `#if 1` in `client.c` but `#if 0` in `server.c` ?

Comment (lurch) [reply pico_w/bt/standalone/client.c:274]: Fair enough :slightly_smiling_face:

Comment (kilograham) [reply pico_w/bt/standalone/client.c:274]: yeah, i also found it a bit odd(confusing); i'm planning to make some comment tweaks when i merge

**Comment by peterharperuk** [comment]
The changes look ok to me

**Pull request merged**